Form/Type/EmbedType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Codete\FormGeneratorBundle\Form\Type;
+
+use Codete\FormGeneratorBundle\FormGenerator;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+
+class EmbedType extends AbstractType
+{
+    /**
+     * @var FormGenerator $formGenerator
+     */
+    private $formGenerator;
+
+    /**
+     * @param FormGenerator $formGenerator
+     */
+    public function __construct(FormGenerator $formGenerator)
+    {
+        $this->formGenerator = $formGenerator;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $class = $options['class'];
+        $this->formGenerator->populateFormBuilder($builder, new $class, $options['view'], isset($options['context']) ? $options['context'] : []);
```

Comment:
Putting new instance of `$model` is rather unexpected as underlying modifiers et al will work on empty `$model` instead of edited one, no? Offhand `new $class` is very optimistic assumption that can easily break if ctor has required arguments - `https://github.com/doctrine/instantiator` helps in such situations.